### ISSUE TITLE
Question: what minimum permission set is needed?

### ISSUE THREAD

**Issue by lvanvugt** [HodorNV/ALOps-External-Deployer#2]
**Question**
What minimum BC permission set is needed for the user that will by used by the ADO release pipeline to install my app through the external deployer?

**Comment by waldo1001** [comment]
Hi Luc - not sure - I'd say: same as a user that needs to be able to install an app with PowerShell, because essentially, that's what's happening.  

Is there an error you're hitting?

**Comment by lvanvugt** [comment]
Nope, no error. Just no sure what a users needs, in your words, _to be able to install an app with PowerShell_.
